meta.json

```diff
@@ -0,0 +1,18 @@
+{
+    "module_id": "viam-labs:pinctrl",
```

Comment:
as far as i can tell this isnt on the registry yet, so just leaving this as a placeholder for testing until the pi5 is no longer in the repo

Reply:
can it go into viam-hardware testing to make it a bit more private? people in viam have access to viam-labs modules and I don't want it proliferating.

Reply:
this is never getting uploaded so we can do whatever we want lol